data/json/items/tools.json

```diff
@@ -10797,5 +10797,30 @@
     "ammo": "NULL",
     "revert_to": "null",
     "use_action": "LADDER"
+  },
+  {
+    "id": "iceaxe",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "dark_gray",
+    "name": "ice axe",
+    "description": "This is an ice axe with hammer on its head, a multi-purpose hiking and climbing tool used by mountaineers.  It is sturdy enough, so you can use it to pry open closed doors or lift manhole covers.",
+    "price": 5000,
+    "material": "steel",
+    "weight": 500,
+    "volume": 3,
+    "bashing": 14,
+    "cutting": 3,
+    "flags" : ["STAB", "DURABLE_MELEE"],
```

Comment:
I had in mind this information from JSON_FLAGS.md: "```STAB``` Deals stabbing damage, with a moderate chance of getting stuck."
That's just what I wanted - have a stabbing damage and a chance to stick.